src/server/handlers.ts

```diff
@@ -230,11 +230,11 @@ const messageCounts: {
   [id: string]: number;
 } = {};
 
-export const SendMessageHandler = (socket: Socket) => {
+export const SendMessageHandler = (io: Server, socket: Socket) => {
```

Comment:
why are you adding io as an argument?

Reply:
This SendMessageHandler didn't get triggered when we call the NEW_EVENT_SEND_MESSAGE event on the client side. I assumed that the reason is, in each handler, io and socket are being passed as parameters, but this function was expectiong only socket as a parameter. When I added io, it started working correctly.

![Screenshot (215)](https://github.com/Dun-sin/Whisper/assets/97130553/ed673e6b-e3b4-430b-8713-77efec418081)

Reply:
makes sense, that means all the functions need to have that io as an argument